manage_people/views.py

```diff
@@ -656,3 +658,21 @@ def lti_key_error_response(request, key_error_exception):
     user_error_message = settings.MANAGE_PEOPLE['MSGS']['lti_request']
     return render(request, 'manage_people/error.html',
                   context={'message': user_error_message}, status=400)
+
+
+def _get_people_in_list_query(user_id_list):
```

Comment:
Would it make sense to put this as a method in icommons_common at this point or even in the local utils file? Not a blocker but just a thought as we are using the same method in a bunch of places at this point

Reply:
Yeah; I thought about putting it in a local utils file but there wasn't one already in this project and I'm hoping that this code doesn't have to stick around for long. If we end up needing to keep this long-term I'll factor it out in to a central library.